examples/jenkins/pipeline/README.md

```diff
@@ -65,7 +67,57 @@ jenkins template represented by jenkinstemplate.json by running these commands a
 
         $ oc start-build sample-pipeline
 
-    Jenkins will: create an instance of the sample-pipeline job, launch a slave, trigger a build in openshift, trigger a
-deployment in openshift, and tear the slave down.
+    Jenkins will: create an instance of the sample-pipeline job, launch a slave, trigger a build in openshift, trigger a deployment in openshift, and tear the slave down.
 
     If you monitor the pods in your default project, you will also see the slave pod get created and deleted.
+
+
+## Pipeline of Pipelines Example
+
+The `mapsapp-pipeline.yaml` template contains a pipeline that instantiates other pipelines and runs them.
+It shows how more than one pipeline can be launched in parallel and how a single Jenkins pipeline
+can work with multiple source code repositories.
+
+To run this example:
+
+1. Ensure that you have a running OpenShift environment as described in the basic example
+2. Create a new project for your pipeline:
+   ```
+   oc new-project mapsapp
+   ```
+
+3. To instantiate the pipeline via the Web Console, create the template from this directory first:
+   ```
+   oc create -f mapsapp-pipeline.yaml
+   ```
+
+4. Go to the Web Console, navigate to the project created above, and click on *Add to project* -> *Uncategorized* -> *mapsapp-pipeline*
+5. Modify the URLs and References of the sample repositories if you have created your own forks.
+6. Click on *Create*
+7. Navigate to *Builds* -> *Pipelines*
+8. Click on *Start Pipeline*
+
+The pipeline will instantiate other pipelines and those will in turn instantiate OpenShift objects. 
```

Comment:
it's going to take some time for jenkins to come up before the pipeline will start, probably worth warning users about that so they aren't immediately wondering why it's not running.